diff --git a/dist/utils/checkPackage.util.js b/dist/utils/checkPackage.util.js
index 9317518f9cc4d70542c526dd27482c50885fb9b4..d50b0a49c5bb8b5ddd8df0a96cd1cb5996be9f79 100644
--- a/dist/utils/checkPackage.util.js
+++ b/dist/utils/checkPackage.util.js
@@ -57,7 +57,7 @@ function checkPackages(packageNames, reason) {
     if (missingDependenciesNames.length) {
         logger.error(MISSING_REQUIRED_DEPENDENCY(missingDependenciesNames, reason));
         logger_service_1.Logger.flush();
-        process.exit(1);
+        // process.exit(1);
     }
     return packages.map((pkg) => pkg.pkg);
 }
